.gitignore

```diff
@@ -163,4 +163,8 @@ cython_debug/
 *.ppm
 
 # vector store
```

Comment:
Could you do me a favour and add `.idea` because of pycharm

Reply:
Sure, no problem. I added it in the latest commit.